### PR TITLE
Includes gifts in MyForest stats for country/project

### PR DESCRIPTION
**Pull request by mohitb35**
- Adds gifts in calculation of country + project stats for My Forest
- Corrects a minor typo in label for project stats

## Summary by CodeRabbit

- **New Features**
	- Introduced new statistics on gifts and contributions by country and project in the "My Forest" section, providing users with insights into the global impact of their contributions.

### CODE REVIEW

**Review by mariahosfeld — COMMENTED** [review]
Functionality looks good to me.

**Comment by norbertschuler** [comment]
Thank you for asking for a review, but as I do not know the details of the backend database modelling, I cannot confirm your queries are correct and all the logic is valid. Better ask @jmiridis for a review of this.

File: src/server/procedures/myForest/stats.ts [hunk 1]
```diff
@@ -67,13 +67,62 @@ export const stats = procedure
         JOIN profile p ON g.recipient_id = p.id
         WHERE p.guid = ${profileId}
       `;
+    // console.log('giftData:', giftData);
+
+    const giftMetadata = await prisma.$queryRaw<CountryProjectStatsResult[]>`
+			SELECT (metadata->>'$.project.id') AS projectId, (metadata->>'$.project.country') AS country
+			FROM gift g 
+			JOIN profile p ON g.recipient_id = p.id
+			WHERE p.guid = ${profileId} AND g.purpose IN ('trees', 'conservation')
```
Comment (norbertschuler) [inline src/server/procedures/myForest/stats.ts:76]: Is this enough to limit the results to valid entries, e.g. do we have to exclude columns with `AND g.deleted_at IS NULL` like for contributions?

Comment (mohitb35) [reply src/server/procedures/myForest/stats.ts]: I've added that in, thanks for letting me know.

**Review by jmiridis — APPROVED** [review]

**Pull request merged**